### ISSUE TITLE
Using your own evaluation and training set

### ISSUE THREAD

**Issue by Ashar2shahid** [GoogleCloudPlatform/cloudml-samples#180]
I would like to use my own set of images for classification: The current method i see is changing the input_path in both training and eval:
`python trainer/preprocess.py \
  --input_dict "$DICT_FILE" \
  --input_path "gs://cloud-ml-data/img/flower_photos/eval_set.csv" \
  --output_path "${GCS_PATH}/preproc/eval" \
  --cloud`

and replacing it with my own csv file pointing to all the images. As it stands that is a very cumbersome process and I would appreciate if someone could help me simplify it.

**Comment by dizcology** [comment]
@Ashar2shahid Please clarify which step you found cumbersome:

1. do you already have the images on GCS?
2. do you have correctly formatted csv files that point to your images?

**Comment by Ashar2shahid** [comment]
I have the images uploaded to GCS into different directories based on their labels eg:
` ------images
-----------------car
-----------------bus
-----------------bike`
with 100 images in each folder. Now creating a csv file for this is as simple as 
`gsutil ls gs://path/to/images/car > car.csv` which creates a csv file with the path to cars but then i have to do this for every folder and also attach the label to the second column and then combine them into one csv file. Doing this for 100 or more folders seems tiresome is there a faster alternative?

**Comment by NazerkeT** [comment]
@Ashar2shahid, If I am not mistaken you can do all these stuff, without this tiresome alternative. Just you need some quick hacking methods. 
1) For example, in mac, you can extract the images' name inside a folder using `ls > contents.txt ` command. 
2) Write some python scripts to combine images' name with the corresponding source folder, then write them into one common csv.
3) You can split them into eval and train sets using the sklearn train_test_split method.
BTW, after the above splitting, you have to merge X_train and y_train, X_test and y_test respectively, using csv module element, for example. 

Good luck!

**Comment by andrewferlitsch** [comment]
@puneith @dizcology The user here is using a typical directory dataset layout (which we seen often in an on-prem scenario). Someone could write a shell script (or python) that could do the command line construction of the CSV file.

Changing from bug to enhancement.

**Comment by nnegrey** [comment]
Going to close this bug as it seems stale, but please feel free to reopen.
